interface/i18n/mist.en.i18n.json

```diff
@@ -83,6 +83,10 @@
                 "win32": "Go to your Internet Time Settings in your system preferences and check the box. See this guide for details: http://www.guidingtech.com/3119/windows-clock-sync/",
                 "linux": "To enable a time sync server install \"ntp\" via \"apt-get install ntp\".",
                 "darwin": "To enable time sync, open the time preferences and check \"Set the time and date automatically\"."
+            },
+            "legacyChain": {
+                "title": "Legacy chain detected",
+                "description": "Your node is currently on the unsupported ethereum classic chain. To use this chain use tools provided by the ethereum classic project at\nhttp://ethereumclassic.github.io.\n\nTo revert to the main ethereum chain follow the tutorial here:\nhttp://github.com/ethereum/mist/releases/0.8.2"
```

Comment:
* Please capitalize, i.e. "Ethereum Classic"
* Use HTTPS: https://ethereumclassic.github.io/
* _To revert...[please] follow the tutorial..._
* Use HTTPS: https://github.com/ethereum/mist/releases/0.8.2